### PR TITLE
Voa 549

### PR DESCRIPTION
**Pull request by Painyjames**

### CODE REVIEW

File: app/controllers/SaveForLater.scala [hunk 1]
```diff
@@ -47,21 +48,51 @@ object SaveForLater extends FrontendController {
...
-          val expiryDate = LocalDate.now.plusDays(playconfig.S4L.expiryDateInDays)
-          val email = sum.customerDetails.flatMap(_.contactDetails.email)
-          EmailConnector.sendEmail(sum.referenceNumber, sum.addressVOABelievesIsCorrect.postcode, email, expiryDate) map { _ =>
-            Ok(views.html.savedForLater(sum, pw, expiryDate))
+      case Some(doc) => {
+        val sum = SummaryBuilder.build(doc)
+        val expiryDate = LocalDate.now.plusDays(playconfig.S4L.expiryDateInDays)
+          if (doc.saveForLaterPassword.isDefined) {
+            playconfig.SaveForLater(doc.saveForLaterPassword.get)(hc)(doc, hc).flatMap { pw =>
+              audit(sum, pw)
```
Comment (rhubner) [inline app/controllers/SaveForLater.scala]: Audit password?

File: project/MicroService.scala [hunk 2]
```diff
@@ -27,14 +29,15 @@ trait MicroService {
   lazy val compileScalastyle = taskKey[Unit]("compileScalastyle")
 
   lazy val microservice = Project(appName, file("."))
-    .enablePlugins(Seq(play.sbt.PlayScala,SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin) ++ plugins : _*)
+    .enablePlugins(Seq(play.sbt.PlayScala,SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory) ++ plugins : _*)
```
Comment (rhubner) [inline project/MicroService.scala]: What is ```SbtArtifactory``` ?

Comment (Painyjames) [reply project/MicroService.scala]: it's a plugin needed for the new build system:
https://confluence.tools.tax.service.gov.uk/display/PLATOPS/Migrate+services+and+libraries+to+new+Build#MigrateservicesandlibrariestonewBuild-Updateyour%60project/plugins.sbt%60file

File: app/controllers/SaveForLater.scala (hunk 1 above)
Comment (Painyjames) [reply app/controllers/SaveForLater.scala]: yeah, was in the code previously but it seems that is not used at all, will remove and see if it still works

Comment (Painyjames) [reply app/controllers/SaveForLater.scala]: it works, removed

**Pull request merged**